pkg/services/alerting/conditions/query.go

```diff
@@ -128,18 +165,28 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange *
 
 		result = append(result, v.Series...)
 
+		queryResultData := map[string]interface{}{}
+
 		if context.IsTestRun {
```

Comment:
Cant we merge `IsTestRun` and `Debug`?

Reply:
I guess we can.

Reply:
Changed my mind. There's different context around testing a rule and debugging a rule.